### PR TITLE
fix: docs strict check errors

### PR DESCRIPTION
**Pull request by pshiu**
## Description

This PR is an attempt to fix the `python setup.py check --restructuredtext --strict` errors.

The error is:

```
(cc) commerce-coordinator % python setup.py check --restructuredtext --strict 
running check
warning: check: Duplicate implicit target name: "added".
```

The warning refers to [line 25](https://github.com/edx/commerce-coordinator/pull/34/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR25) and [line 40](https://github.com/edx/commerce-coordinator/pull/34/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR40) of CHANGELOG.rst having the same implicit target name.

Adding an explicit target name to dedup does not appear to squash the error.

This PR assumes we must avoid deviating from the [standard used by CHANGELOG.rst](https://keepachangelog.com/).

## Additional information

This PR was made as a part of ongoing discussions regarding the best way to do `python setup.py check --restructuredtext --strict` org-wide.

* See:
  * https://github.com/openedx/edx-cookiecutters/pull/204
  * https://github.com/edx/commerce-coordinator/pull/33

## Testing instructions

* On local:
  * [ ] Run `make requirements`
  * [ ] Run `make doc_requirements`
  * [ ] Run `python setup.py check --restructuredtext --strict` and see no errors

### CODE REVIEW

**Comment by pshiu** [comment]
Error is also present in CI: https://github.com/edx/commerce-coordinator/runs/7007906683?check_suite_focus=true#step:6:138

**Comment by robrap** [comment]
> Note: [e5acc07](https://github.com/edx/commerce-coordinator/commit/e5acc0779630b31fef3ef3e7f357747fc41d7e35) (fix: AttributeError due to missing language from code-block directive) cherry-picked from [fb19fe6](https://github.com/edx/commerce-coordinator/commit/fb19fe6881dade39d461160e0a01780566257ba4) in #33 to test @robrap's theory in [openedx/edx-cookiecutters#204 (comment)](https://github.com/openedx/edx-cookiecutters/pull/204#discussion_r903975137).

It seems like that fixed it? Now it just seems like the `added` labels are having an issue. What if you remove the `_` from inside the label name, and just use a space?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -20,17 +20,23 @@ Unreleased
 [0.2.0] - 2021-10-04
 ********************
 
+.. _added_0.2.0:
```
Comment (robrap) [inline CHANGELOG.rst:23]: @pshiu: Separately, if you temporarily update this to `.. _added 0.2.0:` or `.. _anything`, do you get a different result?

**Review by robrap — COMMENTED** [review]
Just wondering if you have proven exactly what is breaking and how to avoid it, even if not using the right permanent solution.

**Comment by pshiu** [comment]
@robrap 

> It seems like that fixed it?

I don't think there ever was any AttributeError in this PR. See link to CI build prior to that commit [here](https://github.com/edx/commerce-coordinator/pull/34#issuecomment-1163303658). The build previous does not have an AttributeError.

Perhaps you were thinking of the AttributeError in https://github.com/edx/commerce-coordinator/pull/33? See [this GitHub actions run](https://github.com/edx/commerce-coordinator/runs/6924148018?check_suite_focus=true). However, (and I forgot this like a fish), I was able to find a fix for the AttributeError (more info: https://github.com/edx/commerce-coordinator/pull/33#issuecomment-1159071755).

I'll try out the other things you mentioned!

**Comment by pshiu** [comment]
This was fixed in upstream https://github.com/openedx/edx-cookiecutters/pull/188.

**Pull request closed without merging**